### PR TITLE
Delete equipments indexes

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: delete_indexed_equipments.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+print("nb indexed tombstoned_equipments = " + get_nb_indexed_tombstoned_equipments())
+print("for a total of = " + str(len(studies)) + " studies")
+print("---------------------------------------------------------")
+if not dry_run:
+    print("Studies indexed equipments and tombstoned deletion processing...")
+    for study in tqdm(studies):
+        delete_indexed_equipments(study['id'])
+    print("End of deletion")
+else:
+    print("nothing has been impacted (dry-run)")
```
Comment (Meklo) [inline delete_indexed_equipments.py:52]: Capitalize ?

File: functions/indexes/indexes.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import requests
+import constant
+
+#
+# @author Hugo Marcellin <hugo.marcelin at rte-france.com>
+# @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+#
+
+def get_nb_indexed_equipments():
+    return requests.get(constant.GET_STUDIES_INDEXED_EQUIPMENTS_COUNT, params={}).text
```
Comment (Meklo) [inline functions/indexes/indexes.py:17]: I reckon params is optional, you can remove it if there's none of it

Comment (sBouzols) [reply functions/indexes/indexes.py]: done

File: delete_indexed_equipments.py (hunk 1 above)
Comment (sBouzols) [reply delete_indexed_equipments.py]: done

**Review by sBouzols — APPROVED** [review]
Test over 10 big studies 1,4M(3M with nested) documents + some tombstoned
-> OK time 56seconds , sleep of 30s for force-merge OK

**Review by jonenst — APPROVED** [review]

**Review by jonenst — APPROVED** [review]

**Pull request merged**